src/plugins/flexibleLayout/components/FlexibleLayout.vue

```diff
@@ -255,7 +271,7 @@ export default {
         container.frames.splice(frameIndex + 1, 0, frame);
         sizeItems(container.frames, frame);
 
-        this.newFrameLocation = [];
+        this.newFrameLocation.splice(0);
```

Comment:
It sounds like we should not be splicing reactive arrays. Let's double check that this change is definitely necessary.